docs/developer-docs/gas-cost.md

```diff
@@ -38,26 +38,28 @@ This documentation will use the following units to measure and calculate the num
 
 ## The ICP reverse gas model
 
-In contrast to other blockchains where users pay for their transactions, ICP charges the smart contract canister for the resources it consumes. This allows developers to provide a smoother user experience, as users are free from tedious tasks such as signing and approving every transaction they perform within the application. This cost model is known as ICP's 'reverse gas model'.
+ICP charges the smart contract canister for the resources it consumes. This allows developers to provide a smoother user experience, as users are free from tedious tasks such as signing and approving every transaction they perform. This cost model is known as ICP's 'reverse gas model'.
```

Comment:
- "where users pay for their transactions" ... I'm wondering whether we should write "end users". Or "the users of the smart contract". Because the dev can also be seen as a user.
- 2 years ago, Dom insisted that we should write "canister smart contract". Let's please stick to it, i.e. not write "smart contract canister"

Reply:
Revised to 'end users'

Revised to 'canister smart contract' and added a note to the Dev Docs Style Guide that this is the correct order of words.